requirements.txt

```diff
@@ -7,4 +8,4 @@ libclang-py3~=3.4.0
 Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
-setuptools>=17.0
+setuptools>=17.0
```

Comment:
Please put this one into a new commit with a nice commit message containing that it screws up your CI ;)